### DISCUSSION TITLE
Is there any possible to have two admin panel in the same time?

### DISCUSSION THREAD

**Discussion by raphaelsoul** [strapi/strapi#6928] [Legacy Topics]
- [ ] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**

While I am wondering if i could setup a totally new admin panel highly customized for doing resource CRUD in `/dashboard` while keeping `/admin` for managing content types and some system level management.

It is good for migrating strapi from old versions to new version in a long maintained project.
But I cannot find any Koa app instance from the source code or some method i can inject the route `/dashboard`

**Comment by raphaelsoul** [comment]
#1465

**Comment by raphaelsoul** [comment]
seems i can do it by calling strapi.app？

**Comment by lauriejim** [comment]
Hello, you can easily create a new custom admin.
The admin panel is a react application using API endpoint to manage everything.
You can easily create your own admin using the same API enpoints.
